playbooks/app_dev/binary_deployment_howto.adoc

```diff
@@ -0,0 +1,197 @@
+---
+---
+= How to do a Simple Binary Deployment to Tomcat in OpenShift
+Eric Sauer <esauer@redhat.com>
+v1.0, 2016-07-13
+:toc: macro
+:toc-title:
+
+toc::[]
+
+Here I am going to walk through the steps to do a simple binary deployment to Tomcat 8 in OpenShift Container Platform. What you need to follow along are:
+
+- A WAR file
+- An OpenShift Environment running 3.1.1 or later
+- The `oc` (OpenShift Client) utility. link:https://docs.openshift.com/enterprise/3.1/cli_reference/get_started_cli.html#installing-the-cli[Install Instructions here].
+
+== Local Setup
+
+Here I have a local directory called `warfile-demo` that just contains my warfile:
+
+[sourc,bash]
+----
+[esauer@localhost ~]$ cd warfile-demo/
+[esauer@localhost warfile-demo]$ ls
+ticket-monster.war
+----
+
+This war file is a jboss-demo sample app we like to use a lot. It would pulled and build from link:https://github.com/jboss-developer/ticket-monster[GitHub].
+
+In order to simplify the URL structure for my application, I'm going to rename my warfile to `ROOT.war`.
```

Comment:
Ticketmonster has an `openshift` profile that can produce a ROOT.war artifact in a `deployments` folder in the project